moz-extensions/src/auth/adapter/PhutilBMOAuthAdapter.php

```diff
@@ -78,19 +82,26 @@ protected function loadOAuthAccountData() {
         $ex);
     }
 
-    // If mfa is disabled in Bugzilla, do not allow login
-    if (
-      PhabricatorEnv::getEnvConfig('bugzilla.require_mfa')
-      && (!isset($result['mfa']) || !$result['mfa'])
-    ) {
-      throw new Exception(
-        pht(
-          'Login using Bugzilla requires multi-factor authentication ' .
-          'to be enabled in Bugzilla. Please enable multi-factor authentication ' .
-          'in your Bugzilla Preferences and try again.'
-        )
-      );
-    }
+    // // If mfa is disabled in Bugzilla, do not allow login
```

Comment:
Heads up that there's commented-out code here.
IMHO, we should delete this code - if we need it in the future, we can peek through VCS to recover it

Reply:
Yeah sorry about that. Forgot to go back and remove it.